htdocs/core/actions_massactions.inc.php

```diff
@@ -58,6 +58,7 @@
     $resaction = '';
     $nbsent = 0;
     $nbignored = 0;
+    $langs->load("main");
```

Comment:
Hum, strange. The main is loaded in main.inc.php file with 

// Load main languages files
if (! defined('NOREQUIRETRAN'))
{
    $langs->load("main");
    $langs->load("dict");
}

For everybody. 
Which key was not translated ? How di you get it ?